packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 0.5.27
+version: 0.5.28
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_plugin_android_lifecycle: ^1.0.0
-  google_maps_flutter_platform_interface: ^1.0.1
+  google_maps_flutter_platform_interface: ^1.0.2
```

Comment:
@ditman, I've added an api to the google_maps_flutter_platform_interface and bumped the version number according to sem versioning. But since this is not hosted anywhere I am getting this issue. `google_maps_flutter depends on google_maps_flutter_platform_interface ^1.0.2 which doesn't match any versions, version solving failed.` To solve this issue, I have to use relative path here, rather than hosted one. And have done the same to run test at my end. Any suggestions regarding this would be helpful.